docs/project/governance.md

```diff
@@ -1,5 +1,5 @@
 (project-governance)=
-# Pyodide Governance and Decision-making
+# Governance and Decision-making
```

Comment:
I think it's clear enough which project it is since Pyodide is already in the logo,
![image](https://user-images.githubusercontent.com/630936/115619349-0c7a6580-a2f4-11eb-9ba0-dee0c9324eba.png)

and this allows to avoid line breaks in the side-menu,
![image](https://user-images.githubusercontent.com/630936/115619421-2156f900-a2f4-11eb-8b19-abcad7567969.png)